Library/Formula/raqm.rb

```diff
@@ -0,0 +1,24 @@
+class Raqm < Formula
+  desc "Library for complex text layout"
+  homepage "https://github.com/HOST-Oman/libraqm"
+  url "https://github.com/HOST-Oman/libraqm/releases/download/v0.1.0/raqm-0.1.0.tar.gz"
+  sha256 "fbccf176ae0ff5aeb73ea3d7a0f2173c432d87a495c10ca2ecdf82323eaa5355"
+
+  depends_on "freetype"
+  depends_on "harfbuzz"
+  depends_on "fribidi"
+  depends_on "glib" => :recommended # for the test suite
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+    system "make", "check"
+  end
+
+  # Raqm does not install any programs
+  test do
+    system "true"
```

Comment:
Please write a test. See [`tinyxml2.rb`](https://github.com/Homebrew/homebrew/blob/599bc69a0e2e20a66d0234b56ac14a65f2e9f7a0/Library/Formula/tinyxml2.rb#L23-L33) for an example of a test for a library.